dissect/target/plugins/os/unix/esxi/_os.py

```diff
@@ -472,7 +472,11 @@ def parse_config_store(fh: BinaryIO) -> dict[str, Any]:
     db = sqlite3.SQLite3(fh)
 
     store = {}
-    for row in db.table("Config").rows():
+
+    if not (table := db.table("Config")):
```

Comment:
Can you make everything a similar style? E.g. don't switch between an early return and an indented loop. I think in this case I slightly prefer the indent over an early return.